app/javascript/components/organization/EditProfileForm.jsx

```diff
@@ -0,0 +1,263 @@
+    formData.append("organization[contact_image]", this.state.contact_selected_file);
+    axios
+      .post("/organizations", formData)
+      .then(function(response) {
+        window.location.href = "/";
+      })
+      .catch(function(error) {
+        console.log(error);
+      });
+  };
+
+  render() {
+    const { organization} = this.state;
+    return {
+      <div className="w-100 h-100 bg-white">
+        <div className="tl fl w-75 ml6 mr6 mt6 mb5 bg-white pa5">
+            <
+            <h1 className="f1 ma0">Edit Profile</h1>
+
+            <h2 className="mt5">Organization Information</h2>
+            <h3 className="mt2">Organization Name</h3>
+            <input
+                className="essay-box bg-light-gray mt1 w-100 pa3"
+                type="text"
+                onChange={this.handleChange("name")}
+                value={this.state.name}
+            />
+            <h3 className="mt3">Website</h3>
+            <input
+                className="dib essay-box bg-light-gray mt1 w-100 pa3 input"
+                type="date"
+                onChange={this.handleChange("website")}
+                value={this.state.link}
+            />
+            <div className="flex justify-between w-100">
+              <div className="dib w-100 mr3">
+                <h3 className="mt3">City</h3>
+                <input
+                    className="dib essay-box bg-light-gray mt1 w-100 pa3 input"
+                    type="text"
+                    onChange={this.handleChange("city")}
+                    value={this.state.city}
+                />
+              </div>
+              <div className="dib w-100 ml3">
+                <h3 className="mt3">State</h3>
+                <input
+                    className="dib essay-box bg-light-gray mt1 w-100 pa3 input"
+                    type="text"
+                    onChange={this.handleChange("state")}
+                    value={this.state.state}
+                />
+              </div>
+            </div>
+            <div className="flex justify-between w-100">
+              <div className="dib w-100 mr3">
+              <h3 className="mt3">Password</h3>
+              <input
+                  className="dib essay-box bg-light-gray mt1 w-100 pa3 input"
+                  type="password"
+                  onChange={this.handleChange("password")}
+                  value={"*****"}
+              />
+              </div>
+              <div className="dib w-100 ml3">
+                <h3 className="mt3">Verify Password</h3>
+                <input
+                    className="dib essay-box bg-light-gray mt1 w-100 pa3 input"
+                    type="password"
+                    onChange={this.handleChange("password_confirmation")}
+                />
+              </div>
+            </div>
+            <div className="dib w-100 ml3">
+              <h3 className="mt3">Add a logo</h3>
+              <input
```

Comment:
Look at Fred's PR on how he does file uploads. It's really pretty and I think we should use that for all file uploads.